prime-router/metadata/schemas/direct/direct-covid-19.schema

```diff
@@ -85,6 +85,15 @@ elements:
   - name: patient_age
     csvFields: [{ name: patientAge}]
 
+  # Using the HHS guidance instead of the FHIR UCUM codes (see https://www.hl7.org/fhir/valueset-age-units.html)
```

Comment:
Is this a hack?     HHS says to use 'yr' for year!  FHIR says to use, 'a'.  Where did THAT come from?